Packages/Notifications/Sources/Notifications/NotificationsListView.swift

```diff
@@ -196,4 +210,16 @@ public struct NotificationsListView: View {
       .frame(height: .layoutPadding)
       .accessibilityHidden(true)
   }
+  
+  private var scrollToTopView: some View {
+    ScrollToView()
+      .frame(height: .scrollToViewHeight)
+      .id(Constants.scrollToTop)
```

Comment:
So, the id could be directly set in the `ScrollToView` you created. You should also move the enum here so you don't duplicate it.

Reply:
Yea, that's an easy change.